agent/internal/docker.go

```diff
@@ -127,6 +133,18 @@ func (d *dockerActor) pullImage(ctx *actor.Context, msg pullImage) {
 			sendErr(ctx, errors.Wrap(err, "error encoding registry credentials"))
 			return
 		}
+	} else if store, ok := d.credentialStores[reference.Domain(ref)]; ok {
+		var creds types.AuthConfig
```

Comment:
weird init because `creds, err := store.get()` shadows `err` in outer scope and linter complain.. newish to go, open to better way around that.